### PR TITLE
SpringCodeGen dateLibrary "java8-localdatetime" option is ignored

### PR DESCRIPTION
**Pull request by nemecec**
Removed 4 lines of code.

Fixes issue #7177.

### CODE REVIEW

**Comment by nemecec** [comment]
> @nemecec thanks for the PR. Please run ./bin/spring-all-petstore.sh to update the Petstore samples if you've not done so.

I ran it now. As those samples do not use `java8-localdatetime` dateLibrary (how could they, it did not work until now), nothing changed in the generated petstore samples.

**Comment by steam0** [comment]
Will this bugfix come in a patch any time soon? Is there any temporary work around to this problem?

**Comment by wing328** [comment]
@steam0 is it correct to say that you've tested this PR and it fixes the problem for you?
```
git checkout -b nemecec-fix-java8-localdatetime-with-springmvc master
git pull https://github.com/nemecec/swagger-codegen.git fix-java8-localdatetime-with-springmvc
```

**Comment by wing328** [comment]
@nemecec I ran some tests with the following setting:
```
{
    "library": "spring-mvc",
    "artifactId": "swagger-spring-mvc-server",
    "dateLibrary": "java8-localdatetime",
    "java8": true
}
```
But got some errors partially shown below:
```
[ERROR] symbol:   variable _valueClass
[ERROR] location: class io.swagger.configuration.CustomInstantDeserializer<T>
[ERROR] /private/tmp/swagger-codegen/samples/server/petstore/spring-mvc/src/main/java/io/swagger/configuration/CustomInstantDeserializer.java:[208,28] cannot find symbol
[ERROR] symbol:   class Instant
[ERROR] location: class io.swagger.configuration.CustomInstantDeserializer<T>
[ERROR] /private/tmp/swagger-codegen/samples/server/petstore/spring-mvc/src/main/java/io/swagger/configuration/CustomInstantDeserializer.java:[208,52] cannot find symbol
[ERROR] symbol:   variable DateTimeUtils
[ERROR] location: class io.swagger.configuration.CustomInstantDeserializer<T>
```
I wonder if you can take a look when you've time.

**Comment by nemecec** [comment]
@wing328 
I tried to reproduce your case and could not (I did encounter issues but those were due to out-of-date version/files).

`CustomInstantDeserializer` is only be generated when you are using `spring-mvc` library and have set `dateLibrary` option to `threetenbp`. Make sure you have rebuilt the code generator from this branch and you do not have any stale source files from some previous generation session.

**Comment by wing328** [comment]
@nemecec thanks. Let me try again with another machine.

**Comment by wing328** [comment]
Now it works for me 👍

**Comment by wing328** [comment]
Added new Spring Petstore samples to CircleCI to cover the async, java8-localdatetime options: https://github.com/swagger-api/swagger-codegen/pull/7468
